geode-logging/src/main/java/org/apache/geode/internal/logging/Loggable.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.internal.logging;
+
+public interface Loggable {
```

Comment:
A comment about what this interface is for would be nice.  Maybe a different name would help, too, like LogWithToString.

Reply:
@kirklund @bschuchardt @demery-pivotal was there any consensus on re-naming this marker interface?  or shall we make a "low-hanging-fruit" Jira ticket...

Reply:
I'd prefer the old EntriesCollection to Loggable.  At least that was somewhat to the point of not wanting to iterate over a huge collection.  I don't think you should open a different JIRA.

Reply:
`EntriesCollection` still exists in `package org.apache.geode.internal.cache` as a marker interface, but is unused...  this could be moved into `geode-logging` and replace `Loggable`
what do y'all think?

Reply:
pushed up this change, btw...